public/stylesheet.css

```diff
@@ -264,16 +264,27 @@ a {
     margin: 0 0 1rem;
   }
 }
-
 .header__list-item {
-  font-family: var(--font-family-stylized);
-  font-size: 1rem;
-  font-weight: 600;
-  margin: 0 0.7rem;
+  height: 55px;
+  width: 55px;
+  background-color: var(--color-background);
+  fill: var(--color-button-active);
 }
-.header__list-item:hover {
-  color: var(--color-button-hover);
-  text-decoration: underline;
+
+.header__list-item:hover,
+.header__list-icon:focus {
+  cursor: pointer;
+  fill: var(--color-background);
+  background-color: var(--color-button-active);
+  border-radius: 50%;
```

Comment:
This used to be `10%` (e.g. in 2fd961def848b63c93c230566ebec660e029d027), why did you change it to `50%`? This surprised me.

First, I don't think the `50%` looks good with the npm link in particular:

![image](https://user-images.githubusercontent.com/3742559/132062789-d111f249-e2b9-4d7f-bdc7-1728dd330c68.png)


More generally speaking, I don't feel like the circle suites the style of the rest of the website. Other elements have slightly rounded corners (e.g. search input, control buttons, cards) [specified in exact pixels](https://github.com/simple-icons/simple-icons-website/blob/cdc99f1be3f68911cbe69b5277daec936e136578/public/stylesheet.css#L24-L25). I would suggest we do the same here for consistency, e.g.:

```suggestion
  border-radius: var(--border-radius-large);
```

What do you think @jorgeamadosoria?

Reply:
I don't have strong opinions one way or the other. I put 50% to make the background round. If you prefer it at 6px, that's fine too.

Reply:
If you think `50%` is the way to go, please stick with that and we ship it that way :slightly_smiling_face:

Reply:
no, no, it's fine. Like I said, I don't have a horse in this race. Let's go with yours.

Reply:
I already committed, even.